definition-map.js

```diff
@@ -140,6 +140,11 @@ const QUERIES = {
         'Flag to randomly order questions and choose a subset of questions to use.',
     },
   },
+  LoadAssessmentAttempts: {
+    metadescription: 
+      'The Load Assessment Attempts endpoint returns all Assessment Attempts within the school. This query requires an authToken header to set a manager user for the operation.',
+      args: {},
```

Comment:
Think you may want to `endDate` and `startDate` here, with descriptions similar to here: https://api.thoughtindustries.com/#list-assessment-attempts

The others (like `userId` and `courseId`, etc) should be covered by the `augment-schema.js` script: https://github.com/thoughtindustries/helium-graphql/blob/d0d92187ea45d2782e3c50f4731e1b546b251d80/augment-schema.js#L81-L92

Reply:
@adam-weinberg-ti Yeah that's my bad, didn't communicate that clearly. My intention was to add them to `definition-map.js` for that specific query, for the reasons you mention. My linking the `augment-schema.js` block there was only to show you how the `userId` and `courseId` args would automatically be filled in as it may be handy to know in the future.